### PR TITLE
add fish-specific install completion

### PR DESCRIPTION
**Pull request by gursheyss**

### CODE REVIEW

File: sce.sh [hunk 1]
```diff
@@ -102,9 +102,14 @@ fi
...
+    if [ -n "$FISH_VERSION" ]; then
+        # Fish shell detected
+        echo "function sce; bash $(pwd)/sce.sh \$argv; end"
+        exit 0
+    fi
+    # For other shells (Bash, Zsh, etc.)
     echo "# for the sce dev tool"
     echo "alias sce=\"$(pwd)/sce.sh\""
-    echo ""
     exit 0
```
Comment (evanugarte) [inline sce.sh:113]: undo this change

Comment (gursheyss) [reply sce.sh]: done

**Review by evanugarte — APPROVED** [review]
